apps/docs-app/docs/features/routing/content.md

```diff
@@ -67,6 +67,7 @@ To get a list using the list of content files in the `src/content` folder, use t
 import { Component } from '@angular/core';
 import { RouterLink, RouterOutlet } from '@angular/router';
 import { injectContentFiles } from '@analogjs/content';
+import { InjectContentFilesFilterFunction } from '@analogjs/content/lib/inject-content-files';
```

Comment:
This shouldn't be a deep import. Instead it should be added to the public API if it's going to be used this way.

Reply:
The `InjectContentFilesFilterFunction` was referenced in the example code, so I added this line. 

https://analogjs.org/docs/features/routing/content#:~:text=InjectContentFilesFilterFunction%3CPostAttributes%3E

If this is not needed, I'll remove this line,
Otherwise, I modify the code to add `InjectContentFilesFilterFunction` to the public API.
Which would be better?

thanks.

Reply:
I would say inlining the filter function in the example would be sufficient for now. We can add it to the public API later if necessary.

Reply:
Thank you for your feedback!
I followed your advice and changed the code. 135298e